docs/conf.py

```diff
@@ -167,5 +167,6 @@
      'Miscellaneous'),
 ]
 
-
-
+language = 'es' # language supported
```

Comment:
es is Spanish not english.  You want 'en'

Reply:
Ok, I have fixed it.